src/Tribe/Assets.php

```diff
@@ -98,9 +93,9 @@ public function register_in_wp( $assets = null ) {
 			foreach ( (array) $asset->action as $action ) {
 				// Enqueue the registered assets at the appropriate time.
 				if ( did_action( $action ) > 0 ) {
-					$this->enqueue();
+					$this->enqueue( $asset->slug );
 				} else {
-					add_action( $action, [ $this, 'enqueue' ], $asset->priority );
+					add_action( $action, [ $this, 'enqueue' ], $asset->priority, 0 );
```

Comment:
:no_entry_sign: Short array syntax is not allowed

<sub>hash: b1ec43c48a272bc1bb2a67d60079fa0d</sub>